can-route-pushstate.js

```diff
@@ -263,43 +254,42 @@ canReflect.assign(PushstateObservable.prototype, {
 	},
```

Comment:
`SimpleObservable` 's default [getter](https://github.com/canjs/can-simple-observable/blob/master/can-simple-observable.js#L56-L59) has similar functionality.
```js
get: function(){
	ObservationRecorder.add(this);
	return this._value;
},
```

Is there a point where this._value **shouldn't** or **isn't** the same as `location.pathname + location.search`?

Reply:
There's the possibility that the observable might be desynced from the URL.